libs/sdk-flutter/pubspec.yaml

```diff
@@ -1,31 +1,31 @@
 name: breez_sdk
 description: A new flutter plugin project.
-version: 0.2.5
+version: 0.3.0
 publish_to: none
 
 environment:
-  sdk: ">=2.19.6 <3.0.0"
-  flutter: ">=3.7.12"
+  sdk: '>=3.0.0 <4.0.0'
+  flutter: ">=3.10.0"
```

Comment:
As we publish a Sdk, keeping the minimum requirement as low as possible is interesting.
So, is it necessary to increase the **minimum** version?

Reply:
Yes, these are the minimum necessary versions.
In Dart 3.0, null safety is built in; you can not turn it off or have a middle-ground, e.g; no `--no-sound-null-safety` flag, and minimum Dart requirement has been upgraded to 3.0 as of Flutter 3.10.0.

Reply:
No, these are the minimum necessary versions due to `flutter_rust_bridge` requirements
`flutter_rust_bridge >=1.76.0 which requires SDK version >=3.0.0 <4.0.0`

which can be downgraded but I think we should move onto Flutter 3.10 & Dart 3.

Reply:
I see; thanks for checking out @erdemyerebasmaz and congrats; that was a hard work